deep-learning/_posts/2017-09-28-DESIRE.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: review
+title: "DESIRE: Distant Future Prediction in Dynamic Scenes with Interacting Agents"
+tags: deep-learning VAE RNN CNN
+author: "Pierre-Marc Jodoin"
+cite:
+    authors: "Namhoon Lee, Wongun Choi, Paul Vernaza, Christopher B. Choy, Philip H. S. Torr, Manmohan Chandraker"
+    title:   "DESIRE: Distant Future Prediction in Dynamic Scenes with Interacting Agents"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1704.04394.pdf"
+---
+
+## Summary
+
+The authors propose a model called *DESIRE* whose goal is to predict future locations of objects in multiple scenes.  The system takes into account the past motion history as well as the scene context to account for interactions among the agents.
+
+![](/deep-learning/images/DESIRE/sc01.png)
+
+As mentioned by the authors (X are the past locations of each object, I is the image at time t, and Y is the future locations of each object), 
+
+> > ```Learning a deterministic function f that directly maps {X, I} to Y will under-represent potential prediction space and easily over-fit to training data. Moreover, a naively trained network with a simple loss will produce predictions that average out all possible outcomes.```
```

Comment:
Please remove the quotes. To quote something you only need the `>`. Example:

    >Vive le Québec libre!